lib/segment/src/fixtures/payload_context_fixture.rs

```diff
@@ -197,15 +197,17 @@ pub fn create_struct_payload_index(
     let mut index = StructPayloadIndex::open(payload_storage, id_tracker, path).unwrap();
 
     index
```

Comment:
There are no `PayloadSchemaType::Text` mention. Maybe it's forgotten

Reply:
Extended this test